lib/Controller/PageController.php

```diff
@@ -152,11 +181,34 @@ public function duplicateSession(): Response {
 	#[NoCSRFRequired]
 	#[PublicPage]
 	#[BruteForceProtection(action: 'talkRoomToken')]
+	#[BruteForceProtection(action: 'callUser')]
 	#[UseSession]
 	public function index(string $token = '', string $callUser = '', string $password = ''): Response {
 		$bruteForceToken = $token;
 		$user = $this->userSession->getUser();
 		if (!$user instanceof IUser) {
+			if ($token === '') {
+				try {
+					$this->limiter->registerAnonRequest(
+						'create-anonymous-conversation',
+						5, // Five conversations
+						60 * 60, // Per hour
+						$this->request->getRemoteAddress(),
+					);
+				} catch(IRateLimitExceededException) {
+					return new TooManyRequestsResponse();
+				}
+
+				$room = $this->createPrivateRoom($callUser);
```

Comment:
- [ ] Need to check if the user enabled it before this action @nickvergessen 
- [ ] Need to check if the admin allowed it before this action @nickvergessen

Reply:
> Need to check if the user enabled it before this action

Implemented now, but requires https://github.com/nextcloud/server/pull/41055